tests/fixture_plugins/orders_fixtures/order_store_service_fixtures.py

```diff
@@ -40,8 +43,18 @@ def store_pacbio_order_service(base_store: Store, lims_api: MockLimsAPI) -> Stor
 
 @pytest.fixture
 def store_metagenome_order_service(
-    base_store: Store, lims_api: MockLimsAPI
+    base_store: Store, lims_api: MockLimsAPI, helpers: StoreHelpers
 ) -> StoreMetagenomeOrderService:
+    metagenome_application: Application = helpers.ensure_application_version(
+        store=base_store, application_tag="METPCFR030"
+    ).application
+    metagenome_application.order_types = [OrderType.METAGENOME]
+    metagenome_application: Application = helpers.ensure_application_version(
+        store=base_store, application_tag="METWPFR030"
+    ).application
+    metagenome_application.order_types = [OrderType.METAGENOME]
+    customer = base_store.get_customer_by_internal_id("cust000")
+    helpers.ensure_user(store=base_store, customer=customer)
```

Comment:
I don't think it is in the scope of this fixture to add tings to the store. I would create a store fixture with all these changes and include it as a parameter to this fixture